pages/about/sponsors.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: US-RSE Sponsors
+permalink: /about/sponsors/
+menubar: about
+set_last_modified: true
+---
+
+US-RSE is currently sponsored in large part from a grant by the Alfred P. Sloan Foundation
+
+<br>
+
+<img src="{{ site.baseurl }}/assets/img/Sloan-Logo-Gold-Blue.png" alt="Alfred P. Sloan Foundation Logo" style="width:500px;">
```

Comment:
Maybe this image could be a link to the Foundation.

Reply:
Depending on what you're going for with the manual sizing (the image is nearly three times wider in pixels), I wonder if it makes more sense to express the width in `em` or % width or somesuch, to respond better to variations in users' pane dimensions and browser DPI. Not a showstopper, might just be worth trying a couple of options and resizing the browser to see how they go, maybe what you have here is already best.

Reply:
I agree with the % approach and center. I fully admit I did a "quick, make it work" approach. I'm sure a better way to scale/position would be better. Don't suppose you have one off the top of your head you could propose?

Reply:
I think this is a good "make it work" start and unless someone wants to update it today, it's fine.

Reply:
Alas I've just moved house (tonight is my first in the new, doing this off cell hotspot!) so exciting times here, I'd be fine with your doing something expedient by setting left and right margins to `auto` and using the `width` attribute on `img`. More proper might be to use CSS `object-fit` and such and I bet our standard CSS has some class you can use that'll center nicely but off-hand I don't recall how to do either and I don't want to obligate you to find out!